packages/checkout/widgets-lib/src/widgets/add-funds/views/AddFunds.tsx

```diff
@@ -230,6 +231,31 @@ export function AddFunds({
   //   console.log('handle review click');
   // }, []);
 
+  useEffect(() => {
+    setAmountInUSD('');
+
+    if (!inputValue || !currentToTokenAddress?.address || !tokens) {
+      return;
+    }
+    const toSquidTokenAddress = currentToTokenAddress?.address === 'native'
+      ? SQUID_NATIVE_TOKEN
+      : currentToTokenAddress?.address;
+
+    const toToken = tokens?.find(
+      (value) => value.address.toLowerCase() === toSquidTokenAddress.toLowerCase()
+        && value.chainId === ChainId.IMTBL_ZKEVM_MAINNET.toString(),
+    );
+
+    if (!toToken) {
+      return;
+    }
+
+    const tokenBalance = Number(inputValue) * toToken.usdPrice;
+    if (tokenBalance > 0) {
+      setAmountInUSD(`USD $${tokenBalance.toFixed(2)}`);
```

Comment:
Please avoid `USD $${tokenBalance.toFixed(2)}` as the amounts format will be added by the translation interpolation.

ie:
`t(`ADD_FUNDS_FIAT_AMOUNT`, { fiatAmount })` where `amount` is `number | string`

this allows the language to set the format for the currency template, also notice rename it to `fiatAmount` as anchor currency could change

Reply:
Done